tests/test_configuration.py

```diff
@@ -140,7 +143,7 @@ def test_keys(config):
 def test_repr(config):
     assert (
         repr(config)
-        == "<Config: 'all_caps', 'debug', 'env_vars', 'general', 'interpolation', 'logging', 'secrets'>"
+        == "<Config: 'all_caps', 'debug', 'env_vars', 'general', 'interpolation', 'logging', 'secrets', 'tasks', 'user_config_path'>"
```

Comment:
How did this change? I don't see a corresponding change in the `config.toml` file

Reply:
`tasks` comes from `process_task_defaults` which is now done in the load configuration function (we weren't using the "full" load function in the old tests). I need to double check why `user_config_path` is automatically present. Maybe it shouldn't be.

Reply:
This was a great call. The user config path one was from our actual test user configuration env vars. I updated everything to be properly isolated.

Reply:
💯 Nice!